focus_converter_base/focus_converter/conversion_configs/gcp/1_pricing_unit_S001.yaml

```diff
@@ -0,0 +1,4 @@
+plan_name: unnest price.unit to PricingUnit
```

Comment:
Any reason why we are going with price.unit instead of usage.pricing_unit ? price.unit is available only in detailed export and goes back only until 2023.
https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/detailed-usage

Reply:
Same as below, was suggest a long time ago.

I can change it to usage.pricing_unit.